ssr/src/page/wallet/tokens.rs

```diff
@@ -132,7 +187,7 @@ pub fn TokenTile(user_principal: String, token_meta_data: TokenMetadata) -> impl
     view! {
         <div class="flex  w-full items-center h-16 rounded-xl border-2 border-neutral-700 bg-white/15 gap-1">
             <a
-                href=format!("/token/info/{root}/{user_principal}?airdrop_amt=100")
+                href=format!("/token/info/{}/{user_principal}?airdrop_amt=100",  root.map(|r| r.to_text()).unwrap_or(info.name.to_lowercase()))
```

Comment:
Can we add functions for these share links? Its really hard to follow which link does what

Reply:
Sure